controllers/machinedeletionremediation_controller_test.go

```diff
@@ -222,7 +223,7 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 						_ = k8sClient.Create(context.Background(), workerNodeMachine) //make sure worker machine will exist - it may be deleted by first run
 						_, reconcileError = reconciler.Reconcile(context.Background(), reconcileRequest)
 						return reconcileError != nil && reconcileError.Error() == mockDeleteFailMessage
-					}).Should(BeTrue())
+					}, 10*time.Second).Should(BeTrue())
```

Comment:
This is a good reminder that the default is 1s only. That's probably why we set both timeout and poll intervall explicitly almost everywhere. Talking about poll intervall: Do you think it's a good idea to increase it as well? The default is 10ms, let's not stress the (fake) API server too much IMHO 😎

Reply:
ah yes, I didn't considered it was so low :+1: